### ISSUE TITLE
plugin stops at line 560 because point is not referenced

### ISSUE THREAD

**Issue by reginaldc** [KIOS-Research/ImportPhotos#29]
I installed the version 1.7 and after adding a folder with pictures and defining an output file I see that the plugin stops at line 560.
I commented out the del instruction and the plugin continues but always I have "No photos were added".

**Comment by reginaldc** [comment]
Update:
After installing:
pillow, osr, exifread, uuid
the plugin runs to completion and the points are generated. The fotos don't show when I click on them with the pointer tool.
Should I install other python modules?

**Comment by EliotWalker** [comment]
I also have this issue - I am on a Mac

**Comment by Mariosmsk** [comment]
Hi @reginaldc , @EliotWalker  . ImportPhotos v1.8 has been released. Check QGIS repository. 
Thanks.

**Comment by Tyrov77** [comment]
It works! Thank you very much!

**Comment by reginaldc** [comment]
Thanks a lot Marios. I tested the new version and it works! Great job!


Reginald Carlier
Deskundige GIS
Gemeente Ingelmunster, Oostrozebekestraat 4, 8770 Ingelmunster
T+32 51 33 74 39www.ingelmunster.be
Schrijf je in op onze nieuwsbrief!
Raadpleeg onze disclaimer 
Van: Marios S. Kyriakou [mailto:notifications@github.com]
Verzonden: woensdag 5 december 2018 18:21
Aan: KIOS-Research/ImportPhotos
CC: Reginald Carlier; Mention
Onderwerp: Re: [KIOS-Research/ImportPhotos] plugin stops at line 560 because point is not referenced (#29)


Hi @reginaldc<https://github.com/reginaldc> , @EliotWalker<https://github.com/EliotWalker> . ImportPhotos v1.8 has been released. Check QGIS repository.
Thanks.
